src/test/code-sample.test.ts

```diff
@@ -0,0 +1,169 @@
+/**
```

Comment:
For consistency, all of our other repos use a `_test.ts` suffix

Reply:
Done.